### PR TITLE
feat: configure native library during installation

### PR DESCRIPTION
**Pull request by achmelo**
Signed-off-by: achmelo <a.chmelo@gmail.com>

### CODE REVIEW

File: common-java-lib-package/src/main/resources/bin/configure.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2019, 2020
+################################################################################
+
+chmod a+x libzowe-attls.so
+extattr +p libzowe-attls.so
```
Comment (jackjia-ibm) [inline common-java-lib-package/src/main/resources/bin/configure.sh:14]: This file will be executed from component root directory, so we need to prefix with `bin/` to find the .so files correctly.

```
chmod a+x bin/libzowe-attls.so
extattr +p bin/libzowe-attls.so
```

Comment (achmelo) [reply common-java-lib-package/src/main/resources/bin/configure.sh]: I've updated paths. Thank you!

File: Jenkinsfile [hunk 2]
```diff
@@ -46,7 +46,7 @@ pipeline {
 
         stage('Install tools') {
             steps {
-                sh 'npm install -g @zowe/cli'
+                sh 'npm install -g @zowe/cli@6.32.2'
```
Comment (pablocarle) [inline Jenkinsfile:49]: I would use here (for now) the v1 lts option

File: gradle.properties [hunk 3]
```diff
@@ -20,7 +20,7 @@ artifactoryPublishingMavenRepo=https://zowe.jfrog.io/zowe/libs-release-local
 artifactoryPublishingMavenSnapshotRepo=https://zowe.jfrog.io/zowe/libs-snapshot-local
 
 # Artifacts version
-version=1.21.4-SNAPSHOT
+version=2.0.0-SNAPSHOT
```
Comment (pablocarle) [inline gradle.properties:23]: I think 2.0.0 has already been released, maybe this will be 2.1.0?

**Review by pablocarle — APPROVED** [review]

**Pull request merged**